cumulus/girderclient.py

```diff
@@ -117,18 +117,15 @@ def _upload_file(self, name, path, parent_id):
 
                 part = fp.read(chunk_size)
 
-                m = MultipartEncoder(
```

Comment:
you should remove line 22

from requests_toolbelt import MultipartEncoder

since you've removed the need for it here.

Reply:
also requests-toolbelt was only used here in the entire repository. So probably should remove it from the requirements.txt file.